File: examples/java/kora-java-crud-submodule/kora-java-crud-submodule-vet-api/src/main/java/io/koraframework/example/submodule/vet/service/VetCircuitBreaker.java
package io.koraframework.example.submodule.vet.service;

import io.koraframework.resilient.circuitbreaker.CircuitBreaker;
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec;

@CircuitBreakerSpec("resilient.circuitbreaker.vet")
public interface VetCircuitBreaker extends CircuitBreaker {

}

File: examples/kotlin/kora-kotlin-crud-submodule/kora-kotlin-crud-submodule-vet-api/src/main/kotlin/io/koraframework/kotlin/example/submodule/vet/service/VetCircuitBreaker.kt
package io.koraframework.kotlin.example.submodule.vet.service

import io.koraframework.resilient.circuitbreaker.CircuitBreaker
import io.koraframework.resilient.circuitbreaker.annotation.CircuitBreakerSpec

@CircuitBreakerSpec("resilient.circuitbreaker.vet")
interface VetCircuitBreaker : CircuitBreaker
